ts/constants.ts

```diff
@@ -18,7 +20,9 @@ export const PRODUCTION_PARAMS: DeploymentParameters = {
     BLOCKS_TO_FINALISE: 7 * 24 * 60 * 4, // 7 days of blocks
     MIN_GAS_LEFT: 10000,
     MAX_TXS_PER_COMMIT: 32,
-    USE_BURN_AUCTION: true
+    USE_BURN_AUCTION: true,
+    DONATION_ADDRESS: "0x00000000000000000000000000000000000000d0",
+    DONATION_NUMERATOR: 7500
```

Comment:
I think it is fine we hardcode the donation address and donation numerator/percentage in the deployment config. It's beneficial to leave them in the commit history.
We could update in the future when we have a clear target to donate.